pkgs/development/libraries/ncurses/default.nix

```diff
@@ -46,6 +46,9 @@ stdenv.mkDerivation rec {
   buildInputs = lib.optional (mouseSupport && stdenv.isLinux) gpm;
 
   preConfigure = ''
+    # These paths end up in the default lookup chain.
+    export TERMINFO_DIRS=/etc/terminfo
+
     export PKG_CONFIG_LIBDIR="$dev/lib/pkgconfig"
```

Comment:
IIUC, then unquoted values in `export a=$b` isn't yet part of POSIX. But bash does already support it (dash doesn't, but we care only for bash in nixpkgs, right? :))

Refs: https://github.com/koalaman/shellcheck/issues/595, http://austingroupbugs.net/view.php?id=351#c943